### DISCUSSION TITLE
Rename Terms & Privacy like Home

### DISCUSSION THREAD

**Discussion by AboveNBeyondCode** [JulianPrieber/littlelink-custom#177] [Feature requests]
Is there a possibility to change the text of terms and privacy like “custom_text_home”?

thus would be a great possibility to use the page on other languages without rewriting all code.

**Comment by JulianPrieber** [comment]
By "text" do you mean the title of the terms page?

**Reply by AboveNBeyondCode** [reply]
Hi
Yes, exactly.
So that I am able to change the phrase terms and privacy to sth else, like „Impressum“ and „Datenschutzerklaerung“ for example.
